foundation/foundation-mda/src/main/resources/templates/deployment/kafka/kafka.values.yaml.vm

```diff
@@ -5,12 +5,14 @@
 replicaCount: 1
 
 hostname: kafka-cluster
+
+  # NB: 1.7.0 is the last version to include kafka support. It is being frozen there while we
+  # align it to use a community image. TODO in https://github.com/boozallen/aissemble/issues/228
 image:
   name: boozallen/aissemble-kafka
   imagePullPolicy: Always
   dockerRepo: ghcr.io/
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: "${versionTag}"
+  tag: "1.7.0"
```

Comment:
S: I think we need to update the migrations in `foundation-upgrade` to skip over updating the tags for these charts.  Might also make sense to throw a note in the release notes about these images being deprecated.

Reply:
Good catch - updated!